### PR TITLE
Rename crds and create release channels

### PR DESCRIPTION
**Pull request by TerryHowe**
Make these names consistent with what is uploaded. Any new versions should be named with versions in them like this.

### CODE REVIEW

File: release/Makefile [hunk 1]
```diff
@@ -24,13 +24,13 @@ run: generate fmt vet manifests
 
 # Install CRDs into a cluster
 install: manifests
-	kubectl apply -f ./config/crds/distro.eks.amazonaws.com_releasechannels.yaml
-	kubectl apply -f ./config/crds/distro.eks.amazonaws.com_releases.yaml
+	kubectl apply -f ./config/crds/releasechannels.distro.eks.amazonaws.com-v1alpha1.yaml
```
Comment (abhay-krishna) [inline release/Makefile:27]: We would also need to rename the actual files referenced here right?

**Comment by TerryHowe** [comment]
Waiting on https://github.com/aws/eks-distro-build-tooling/pull/107 I guess

**Comment by mrajashree** [comment]
@TerryHowe #107 is merged now

**Review by abhay-krishna — APPROVED** [review]
/lgtm

**Pull request merged**